src/opm/common/utility/numeric/calculateCellVol.cpp

```diff
@@ -131,4 +131,11 @@ double calculateCellVol(const std::array<double,8>& X, const std::array<double,8
 }
 
 
-
+/* 
+    Cell volume calculation for a cell from a cylindrical grid, given by the
+    inner and outer radius of the cell, and its spans in the angle and Z.
+*/
+double calculateCylindricalCellVol(const double r_inner, const double r_outer, const double delta_theta, const double delta_z)
+{
+    return M_PI * abs((r_inner*r_inner - r_outer*r_outer) * delta_theta * delta_z) / 360.0;
```

Comment:
Ehhh - why not just (outer^2  - inner^2)?

Reply:
yes, that would be clearer.